data/Artificial_Intelligence.md

```diff
@@ -8,3 +8,4 @@
 | [Perplexity.ai](https://perplexity.ai) | OpenAI GPT-model powered AI chatbot that's connected to the web. Find articles or ask for current info. |
 | [Phind](https://phind.com) | AI chatbot built for developers in a chat-like form. |
 | [Poe](https://poe.com) | Poe is chat-based AI. Uses GPT-3.5. |
+| [Github Copilot](https://github.com/features/copilot) | **GitHub Copilot is free to use for verified students, teachers, and maintainers of popular open source projects.** If you are not a student, teacher, or maintainer of a popular open source project, you can try GitHub Copilot for free with a one-time 30-day trial. |
```

Comment:
We have purposely not added GitHub Copilot to our list as it is not free forever, it is only a free trial.